webapi/src/main/scala/org/knora/webapi/messages/StringFormatter.scala

```diff
@@ -2117,14 +2168,15 @@ class StringFormatter private (
    *                    email address.
    * @return the same optional string.
    */
+  @deprecated("Use validateAndEscapeOptionalEmail(Option[String]) instead.")
   def validateAndEscapeOptionalEmail(
     maybeString: Option[String],
     errorFun: => Nothing
   ): Option[String] = // V2 / value objects
-    maybeString match {
-      case Some(s) => Some(toSparqlEncodedString(validateEmailAndThrow(s, errorFun), errorFun))
-      case None    => None
-    }
+    maybeString.map(validateEmailAndThrow(_).getOrElse(errorFun))
+
+  def validateAndEscapeOptionalEmail(maybeString: Option[String]): Option[String] =
```

Comment:
Consider removing this method and inline the original `validateAndEscapeOptionalEmail` instead.

Reply:
All "optional" methods are gone now 🔥